integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,15 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
This is per container setting and I have verified that after creating container with mem-swappiness 0 for example the value in /sys/fs/cgroup/memory/system.slice/docker-xxx/memory.swappiness changes as expected